datacube_alchemist/transforms.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+
+import numpy
+from datacube import Datacube
+from datacube.virtual import Measurement, Transformation
+from xarray import Dataset, merge
+
+
+class DeltaNBR(Transformation):
+    """Return NBR"""
+
+    def __init__(self):
+        self.output_measurements = {
+            "dnbr": {"name": "dnbr", "dtype": "float", "nodata": -9999, "units": ""}
+        }
+
+    def measurements(self, input_measurements) -> Dict[str, Measurement]:
+        return self.output_measurements
+
+    def compute(self, data) -> Dataset:
+        base_year = data.time.dt.year.values[0] - 1
+        if base_year == 2021:
+            base_year = 2020
+        if base_year == 2012:
+            base_year = 2013
+
+        dc = Datacube()
+        gm_data = dc.load(
+            product="ls8_nbart_geomedian_annual",
```

Comment:
Maybe better to pass in *comparison product name* from the configuration? But, then I'm not sure about how the custom logic for selecting years fits...